transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringEventLoop.java

```diff
@@ -268,7 +278,12 @@ protected void wakeup(boolean inEventLoop) {
         }
     }
 
-    public IovecArrayPool getIovecArrayPool() {
-        return iovecArrayPool;
+    public IovArray iovArray() {
+        // Check if its full and if so submit so we can reuse it.
+        if (iovArray.isFull()) {
```

Comment:
one drawback would be that we reduce submission batch size when a lot of channel writing in the same time especially with a bigger message size, one solution would be: to make the capacity configurable, default iovec array size is only 1024 as far as I know

Reply:
yep I am aware of this... thats why I added some "we may want to adjust this later comment"... That said let me run some benchmarks for now first